statsbestproducts.php

```diff
@@ -191,6 +197,16 @@ public function getData()
         foreach ($values as &$value) {
             $value['avgPriceSold'] = Tools::displayPrice($value['avgPriceSold'], $currency);
             $value['totalPriceSold'] = Tools::displayPrice($value['totalPriceSold'], $currency);
+	    $q = "SELECT count(oret.id_order_return) AS nr FROM "._DB_PREFIX_."order_return AS oret
+LEFT JOIN "._DB_PREFIX_."order_return_detail AS ord ON (oret.id_order_return=ord.id_order_return)
+LEFT JOIN "._DB_PREFIX_."order_detail AS od ON (ord.id_order_detail=od.id_order_detail)  
+LEFT JOIN "._DB_PREFIX_."product_attribute AS pa ON (od.product_attribute_id=pa.id_product_attribute)
+LEFT JOIN "._DB_PREFIX_."order_return_state_lang AS status ON (oret.state=status.id_order_return_state) 
+WHERE status.id_lang=1 
+AND date_add BETWEEN $date_between
+AND od.product_id='" . $value['id_product'] . "';";
```

Comment:
```suggestion
AND od.product_id='" . (int)$value['id_product'] . "';";
```
Thanks to Arnaud from SC

Reply:
Done, thanks @Arnaud